src/unix/unix.c

```diff
@@ -109,7 +109,7 @@ void deliver_fault_signal(u32 signo, thread t, u64 vaddr, s32 si_code)
     };
 
     char *signame = "SIGSEGV";
-    assert(signo == SIGSEGV || signo == SIGBUS || signo == SIGFPE);
+    assert(signo == SIGSEGV || signo == SIGBUS || signo == SIGFPE || signo == SIGILL || signo == SIGTRAP);
```

Comment:
Could we remove this assert and add a default clause to the following switch() statement, with a halt or assert in there?